extensions/ql-vscode/package.json

```diff
@@ -926,10 +922,6 @@
           "command": "codeQL.viewCfg",
           "when": "resourceScheme == codeql-zip-archive && config.codeQL.canary"
         },
-        {
```

Comment:
How come we're removing this command? Should we just remove the when clause?

Reply:
See my comment above, when there's no condition the command doesn't need to be in the `commandPalette` section.